compute_tools/src/spec.rs

```diff
@@ -269,17 +273,21 @@ pub fn handle_roles(spec: &ComputeSpec, client: &mut Client) -> Result<()> {
                 xact.execute(query.as_str(), &[])?;
             }
             RoleAction::Create => {
-                let mut query: String = format!("CREATE ROLE {} ", name.pg_quote());
+                let mut query: String =
+                    format!("CREATE ROLE {} IN ROLE neon_superuser", name.pg_quote());
                 info!("role create query: '{}'", &query);
                 query.push_str(&role.to_pg_options());
                 xact.execute(query.as_str(), &[])?;
 
-                let grant_query = format!(
-                    "GRANT pg_read_all_data, pg_write_all_data TO {}",
-                    name.pg_quote()
-                );
-                xact.execute(grant_query.as_str(), &[])?;
-                info!("role grant query: '{}'", &grant_query);
+                // Remove this when we get rid of web_access
+                if spec_has_webaccess(spec) {
+                    let grant_query = format!(
+                        "GRANT pg_read_all_data, pg_write_all_data TO {}",
+                        name.pg_quote()
+                    );
+                    xact.execute(grant_query.as_str(), &[])?;
+                    info!("role grant query: '{}'", &grant_query);
+                }
```

Comment:
Not sure, but it feels like if we now grant `neon_superuser` (it already has `pg_read_all_data, pg_write_all_data`) to every new role, then we don't need this code at all